notebooks/spec2vec/tutorial.ipynb

```diff
@@ -33,7 +33,7 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 3,
+   "execution_count": 1,
```

Comment:
the token should not be committed here either 
![image](https://user-images.githubusercontent.com/63718157/127297457-bd5813e9-ae49-43d5-b67a-9d1627924975.png)
just commit "my-token" instead